testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -305,10 +306,12 @@ impl TxEmitter {
     }
 
     fn make_client(&self, instance: &Instance) -> JsonRpcAsyncClient {
-        JsonRpcAsyncClient::new(
+        JsonRpcAsyncClient::new_with_client(
             self.http_client.clone(),
-            instance.ip(),
-            instance.ac_port() as u16,
+            Url::from_str(
+                format!("http://{}:{}", instance.ip(), instance.ac_port() as u16).as_str(),
```

Comment:
very small nit: you probably no longer need `as u16` in this context

Reply:
fixed.